flux-middle/src/ty/subst.rs

```diff
@@ -80,13 +80,13 @@ impl TypeFolder for FVarFolder<'_> {
     }
 }
 
-pub(crate) struct BVarFolder<'a> {
+pub struct BVarFolder<'a> {
```

Comment:
This `pub(crate)` is on purpose to try enforcing the `Binders` abstraction.

```suggestion
pub(crate) struct BVarFolder<'a> {
```